int32 beacon_id
int32 tag_id
float64 range
float64 rss
float64 error_estimation